### DISCUSSION TITLE
Is the WIFI SUN2000-<SER#> Modbus independent from the Dongle Modbus?

### DISCUSSION THREAD

**Discussion by henfri** [wlcrs/huawei_solar#368] [General]
I have two devices that would like to access my Inverter via Modbus: My wallbox and Home Assistant.
@wlcrs : You told me that there is no way to use Modbus RTU while still using the App.

Thus, I would like to use the most reliable way for Home Assistant, i.e. the connection to the Inverter Wifi and to use the Dongle for the App-Data into the Cloud and for the Wallbox.

Is that possible, or are the Wallbox and the Inverter Wifi Modbus Connection the same again, so that they would interfere?

Then, I quess, I would need a Modbus Proxy, right? 

I would like to avoid that, to have not additional possible point of failure.

Regards,
Hendrik

**Comment by Starship2Mars** [comment]
I would like to know that too. Same situation. OpenWB is connected via the dongle (LAN) and I want to feed the Inverter integrated WLAN into the LAN… but I don’t know how…
